app.js

```diff
@@ -1,57 +1,415 @@
-var geofencingApiURL = 'https://api.tomtom.com/geofencing/1/';
+var currentTab = "api-key-form";
```

Comment:
Let's stick to using either var or const and let.

Reply:
replaced var with const and let